python/private/bzlmod/label.bzl

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"A small utility to create a label usable internally"
```

Comment:
Yeah, this is one of the ways for us to construct a label that we can pass to repositories created by pip. This would be unneeded if https://github.com/bazelbuild/bazel/issues/19055 was fixed in 7.1.0 and we used it.